src/testing/TestScheduler.ts

```diff
@@ -1,24 +1,23 @@
 import Observable from '../Observable';
-import VirtualTimeScheduler from './VirtualTimeScheduler';
+import VirtualTimeScheduler from '../schedulers/VirtualTimeScheduler';
 import Notification from '../Notification';
 import Subject from '../Subject';
+import ColdObservable from './ColdObservable';
+import HotObservable from './HotObservable';
+import TestMessage from './TestMessage';
+import SubscriptionLog from './SubscriptionLog';
 
 interface FlushableTest {
-  observable: Observable<any>;
-  marbles: string;
   ready: boolean;
   actual?: any[];
   expected?: any[];
 }
```

Comment:
They were hold overs from building/debugging initially I think. The whole thing used to work differently